Explorer/Assets/Scripts/Global/ComponentsContainer.cs

```diff
@@ -27,23 +28,30 @@ public static ComponentsContainer Create()
             var sdkComponentsRegistry = new SDKComponentsRegistry();
 
             // SDK RaycastHit (used only as an element in the list)
-            var raycastHitPool = new ComponentPool<RaycastHit>(defaultCapacity: 100, onGet: c => c.Reset());
+            var raycastHitPool = new ComponentPool.WithDefaultCtor<RaycastHit>(defaultCapacity: 100, onGet: c => c.Reset());
 
             // Add all SDK components here
             sdkComponentsRegistry
-               .Add(SDKComponentBuilder<SDKTransform>.Create(ComponentID.TRANSFORM).WithPool(SDKComponentBuilderExtensions.SetAsDirty).WithCustomSerializer(new SDKTransformSerializer()).Build())
+               .Add(SDKComponentBuilder<SDKTransform>.Create(ComponentID.TRANSFORM)
+                                                     .WithPool(sdkTransform =>
+                                                      {
+                                                          sdkTransform.Clear();
```

Comment:
This is executed onGet not onRelease is that right?

Reply:
right